chain/gen/gen.go

```diff
@@ -229,7 +229,7 @@ func NewGeneratorWithSectorsAndUpgradeSchedule(numSectors int, us stmgr.UpgradeS
 		Timestamp:        uint64(build.Clock.Now().Add(-500 * time.Duration(build.BlockDelaySecs) * time.Second).Unix()),
 	}
 
-	genb, err := genesis2.MakeGenesisBlock(context.TODO(), j, bs, sys, tpl)
+	genb, err := genesis2.MakeGenesisBlock(context.TODO(), j, bs, sys, tpl, false)
```

Comment:
Can we choose the context more precisely?

Reply:
Is Lotus code not Eudico-specfic, I just added the flag so it compiles, I rather l leave it as-is if you don't mind.